helloworld.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!${HOME}/SlowMovie/.venv/bin/python3 
```

Comment:
Thoughts on just removing this entirely? I'm 50/50. If running within the virtual environment with `python3 helloworld.py` it will ignore this line anyway. My hesitation is that we don't know the target filesystem, what if the user decided to install this in `${HOME}/projects/SlowMovie`? Also not sure it will expand ${HOME} properly, it didn't seem to work when I tested it.

Reply:
So I gave this some thought and I agree reverting that change is the way to go. My original intent was to make it easy for someone to use (so have it work out of the box) and avoid the need to source the virtual environment if they were running it interactively without the included service file. I think perhaps a better approach would be to just have the installer add the activation of the venv to the user profile so it happens automatically on login. This would cover the interactive use case w/o the need for a bunch of complex code to test and source the venv if it wasn't sourced. The other option I can think of is to write a small wrapper that would handle sourcing it and instruct people to use that vs. calling `python slowmovie.py`.

Thoughts on if either of those approaches are desirable? Maybe I'm over thinking this and it's better to expect that the end-user can handle sourcing the venv on their own.

Reply:
I think in _most_ cases it's probably correct to think that setting up SlowMovie will be pretty much the only application for the Pi. There is probably the off case - like testing - where it isn't but most people want to set it up and display it somewhere. I think if you can add activating the environment to the profile that would work. More DIY users will know how to remove it or are installing manually anyway.

Reply:
Cool, I agree in most cases that would be what people do and making the default behavior "just work" would be the right thing to do for users. 

I went ahead and revised the installer to add logic to modify the installing users .profile file to handle sourcing the venv automatically on login. On a stock Raspberry PI OS system this is the file sourced for user logins - I figure if a DYI user has modified their shell from the OS default they are probably capable of handling sourcing it themselves.

The only other thing I can think of changing is a cosmetic thing. Right now the venv is called "venv" so once it's sourced the user shell looks like this:

_(.venv) jon@CodeVM:~$_ 

Since the project is named **SlowMovie** would it make more sense to call it "SlowMovie" so it looked like this instead?

_(.SlowMovie) jon@CodeVM:~$_

It would be a simple change to make but I'd like your input on what it should be.

Reply:
I think it's a nice touch so it doesn't look so generic. Getting this up and running should be as non-technical as possible so anything that can help with that is good.

Reply:
Done & tested. My latest commit changes it to be "**SlowMovie**"